development/docker-compose.yml

```diff
@@ -89,6 +89,7 @@ services:
     image: selenium/standalone-firefox:4.1.2
     ports:
         - "4444:4444"
+        - "15900:5900"
```

Comment:
What is this added for?

Reply:
```
- Also map VNC port for Selenium container (5900/tcp) to 15900/tcp on host
```

Reply:
Was this just something you discovered in Selenium best practices? I'm fine with it but didn't see on the surface why this would be required for Dynamic Groups of Groups

Reply:
It's not required, it's just so that when debugging integration tests one can VNC into the Selenium container and watch what happens. Very handy when trying to figure out why an integration test keeps failing.

Reply:
👍